tests/components/heos/test_media_player.py

```diff
@@ -0,0 +1,187 @@
+"""Tests for the Heos Media Player platform."""
+from pyheos import const
+
+from homeassistant.components.heos import media_player
+from homeassistant.components.heos.const import DOMAIN
+from homeassistant.components.media_player.const import (
+    ATTR_MEDIA_ALBUM_NAME, ATTR_MEDIA_ARTIST, ATTR_MEDIA_CONTENT_ID,
+    ATTR_MEDIA_CONTENT_TYPE, ATTR_MEDIA_DURATION, ATTR_MEDIA_POSITION,
+    ATTR_MEDIA_POSITION_UPDATED_AT, ATTR_MEDIA_SHUFFLE, ATTR_MEDIA_TITLE,
+    ATTR_MEDIA_VOLUME_LEVEL, ATTR_MEDIA_VOLUME_MUTED,
+    DOMAIN as MEDIA_PLAYER_DOMAIN, MEDIA_TYPE_MUSIC, SERVICE_CLEAR_PLAYLIST,
+    SUPPORT_NEXT_TRACK, SUPPORT_PAUSE, SUPPORT_PLAY, SUPPORT_PREVIOUS_TRACK,
+    SUPPORT_STOP)
+from homeassistant.const import (
+    ATTR_ENTITY_ID, ATTR_FRIENDLY_NAME, ATTR_SUPPORTED_FEATURES,
+    SERVICE_MEDIA_NEXT_TRACK, SERVICE_MEDIA_PAUSE, SERVICE_MEDIA_PLAY,
+    SERVICE_MEDIA_PREVIOUS_TRACK, SERVICE_MEDIA_STOP, SERVICE_SHUFFLE_SET,
+    SERVICE_VOLUME_MUTE, SERVICE_VOLUME_SET, STATE_IDLE, STATE_PLAYING,
+    STATE_UNAVAILABLE)
+
+
+async def setup_platform(hass, config_entry, players):
+    """Set up the media player platform for testing."""
+    hass.config.components.add(DOMAIN)
```

Comment:
We should make sure to actually set up the component since it has a config section that should be validated. Use `async_setup_component` instead of forwarding the entry.

Reply:
Done.  Changing this throughout did increase test from 1s to over 5s.